api/src/api/storage.ts

```diff
@@ -27,7 +27,11 @@ export interface Storage {
 
   getWikiConfig(): WikiConfig;
 
-  getPageRestURL(page: string, syntax: string): string;
+  getPageRestURL(
+    page: string,
+    syntax: string,
+    revision: string | undefined,
```

Comment:
To be double-checked, but I think this syntax is equivalent, more concise and already present in our code base.
```suggestion
    revision?: string,
```